optimism.tokenlist.json

```diff
@@ -1296,7 +1297,7 @@
       "decimals": 18,
       "logoURI": "https://ethereum-optimism.github.io/data/DAI/logo.svg",
       "extensions": {
-        "baseBridgeAddress": "0x4200000000000000000000000000000000000010",
+        "pgnBridgeAddress": "0x4200000000000000000000000000000000000010",
```

Comment:
THis change makes it so we will have to bump the version number in package.json a major version

Reply:
I guess so yes